.circleci/config.yml

```
@@ -44,6 +44,51 @@ jobs:
           working_directory: ./web
           command: yarn test:unit
 
+  test_e2e:
+    docker:
+      - image: circleci/python:3.6-node-browsers
+      - image: viime/opencpu-base
+        command: |
+          /bin/sh -c "chmod a+w /usr/local/lib/R/site-library && service cron start && /usr/lib/rstudio-server/bin/rserver && apachectl -DFOREGROUND"
+    steps:
+      - checkout
+      - restore_cache:
+          name: Restore Yarn Package Cache
+          keys:
+            - yarn-packages-{{ checksum "e2e-test/package.json" }}
+      - save_cache:
+          name: Save Yarn Package Cache
+          key: yarn-packages-{{ checksum "e2e-test/package.json" }}
+          paths:
+            - ~/.cache/yarn
+      - run:
+          working_directory: ./e2e-test
+          name: Install dependencies
+          command: yarn
+      - run:
+          working_directory: ./e2e-test
+          name: Run lint tests
+          command: yarn run lint
+      - run:
+          working_directory: ./e2e-test
```

```suggestion
          working_directory: ./web
```
This will navigate to the `e2e-test` directory, then run `yarn serve`. `serve` isn't defined in `e2e-test` though, it's defined in `web`. You can see this step failing [here](https://app.circleci.com/pipelines/github/girder/viime/1629/workflows/023da26e-f4c2-41fe-9cba-cb26975775ce/jobs/4121/parallel-runs/0/steps/0-106).

---

Oh I see it now.